### PR TITLE
Feat/analytics 1.13

### LINKED ISSUE #4189: Data Analytics
After consultation with @tomasklim, we want some changes in suite logs:

- [ ] rename event `wallet/created` to something more intuitive, e.g. `select-wallet-type`, and `add-wallet` from modal switch-device rename to `add-hidden-wallet`
- [ ] delete event `desktop-init`: event has no added value, its only unique atributte `DesktopOsVersion` is not used anywhere.

### PR DESCRIPTION
**Pull request by tomasklim**
closes #4189

### CODE REVIEW

File: packages/suite-desktop/src-electron/libs/user-data.ts [hunk 1]
```diff
@@ -39,19 +39,3 @@ export const read = async (directory: string, name: string) => {
         return { success: false, error };
     }
 };
-
-export const getOsType = () => {
-    try {
-        const platform = os.platform();
```
Comment (matejkriz) [inline packages/suite-desktop/src-electron/libs/user-data.ts:45]: `os` is no more used

File: packages/suite/src/actions/suite/analyticsActions.ts [hunk 2]
```diff
@@ -521,22 +509,6 @@ export const init =
                 }),
             );
         });
-
-        // send OS type if isDesktop
-        if (isDesktop()) {
-            let desktopOSVersion = '';
-            const resp = await getOsType();
```
Comment (matejkriz) [inline packages/suite/src/actions/suite/analyticsActions.ts:528]: remove also `async` keyword for the function, there is no more await

File: packages/suite-desktop/src-electron/libs/user-data.ts (hunk 1 above)
Comment (tomasklim) [reply packages/suite-desktop/src-electron/libs/user-data.ts:45]: Fixed and squashed

File: packages/suite/src/actions/suite/analyticsActions.ts (hunk 2 above)
Comment (tomasklim) [reply packages/suite/src/actions/suite/analyticsActions.ts:528]: Fixed and squashed

**Comment by matejkriz** [comment]
@tomasklim could you check the failing e2e test?

**Review by matejkriz — APPROVED** [review]
👍 feel free to rebase fixup and merge

**Comment by tomasklim** [comment]
> @tomasklim could you check the failing e2e test?

I will check, this reminds me, that we should enable tracking analytics from localhost. (there are some dev endpoints) Because right now it is not possible to launch analytics e2e test from localhost and it makes developing of analytics harder.

**Pull request merged**